zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/user/UserStateTest.java

```diff
@@ -109,4 +109,29 @@ void shouldReturnCorrectUserByUsername() {
     assertThat(persistedUserOne.getUsername()).isEqualTo(usernameOne);
     assertThat(persistedUserTwo.getUsername()).isEqualTo(usernameTwo);
   }
+
+  @DisplayName("should update a user")
+  @Test
+  void shouldUpdateAUser() {
```

Comment:
🔧  Let's test all the fields that can get updated to be complete.

Reply:
Good catch, I've changed